### ISSUE TITLE
Authorization Bearer, once authenticated, works even in unrelated sessions

### ISSUE THREAD

**Issue by heymartinadams** [prisma/prisma#2881]
**Describe the bug**
I followed the steps in the [Prisma Docs](https://docs-beta.prisma.io/1.13/use-prisma-api/api-reference/authentication-ghd4/) on Authentication using `secret` and `prisma token`. Once the `token` is generated and sent via the header to the server, however, it appears that I have unlimited access to the server — even if I remove the `Authorization: Bearer __TOKEN__` from the header, and even if I open an incognito browser tab or use a different browser (I’m accessing the server client-side via `window.fetch`).

**To Reproduce**
1. Follow the steps in the [Prisma Docs](https://docs-beta.prisma.io/1.13/use-prisma-api/api-reference/authentication-ghd4/) example to set a prisma `secret`, then generate a `__TOKEN__` using `prisma token`.
2. Client-side, use `window.fetch` to access the server. Be sure to include `Authorization: Bearer __TOKEN__` in the header.
```javascript
window.fetch(__SERVER_URL__, {
  headers: {
    Authorization: 'Bearer __TOKEN__'
  },
  method: 'POST',
  mode: 'cors',
  body: JSON.stringify({
    query: __API__,
    variables
  })
}
```
3. Run the code, so that the token gets sent to the server and you get a response.
4. Re-run the code, but this time without the `Authorization: 'Bearer __TOKEN__'`:
```javascript
window.fetch(__SERVER_URL__, {
  method: 'POST',
  mode: 'cors',
  body: JSON.stringify({
    query: __API__,
    variables
  })
}
```

**Expected behavior**
Shouldn’t step 4 produce an error if it’s run in a different browser or in an incognito tab? I’m still able to access the entire API, even in a different browser or an incognito session.

**Versions (please complete the following information):**
 - OS: `macOS High Sierra 10.13.3`
 - `prisma` CLI: `prisma/1.13.5 (darwin-x64) node-v9.7.1`
 - Prisma Server: `1.10.2`

**Comment by marktani** [comment]
Did you deploy the modified `prisma.yml` after step 1?

**Comment by heymartinadams** [comment]
In step 1, I did:

1.a. Set/change the `secret`
1.b. `prisma deploy`
1.c. Uploaded to server: `now --dotenv .env`
1.d. Generated a token using `prisma token`

Then,
2. Used `window.fetch` with `Authorization: Bearer __TOKEN__`. Works as expected.

3. Then removed `Authorization: Bearer __TOKEN__` from the header, but still works, even in different browsers.

Since I ran the call to the server from `localhost:3000`, does the authorization token enable all calls to the server from that domain, even in different browsers, sessions that don’t have the token in the header?
